content/doc/book/pipeline/docker.adoc

```diff
@@ -0,0 +1,421 @@
+---
+layout: section
+title: Using Docker with Pipeline
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= Using Docker with Pipeline
+
+Many organizations use link:https://www.docker.com[Docker] to unify their build
+and test environments across machines, and to provide an efficient mechanism
+for deploying applications. As of version 2.5, Pipeline has built-in support
```

Comment:
I thought it was clearly indicating Pipeline version 2.5, will reword to avoid confusion